ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -94,10 +95,14 @@ public CommandId distributeStatement(
             String statementString,
             Statement statement,
             Map<String, Object> streamsProperties
-    ) throws Exception {
+    ) throws KsqlException {
         CommandId commandId = commandIdAssigner.getCommandId(statement);
         Command command = new Command(statementString, streamsProperties);
-        commandProducer.send(new ProducerRecord<>(commandTopic, commandId, command)).get();
+        try {
+            commandProducer.send(new ProducerRecord<>(commandTopic, commandId, command)).get();
+        } catch (Exception e) {
+            throw new KsqlException("Could not write the statement into the command topic.", e);
```

Comment:
Should we add the statementString in the exception method so we have a bit more context?

Reply:
Sure, added the statement string to the message.